specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json

```diff
@@ -7535,11 +7544,27 @@
         "modelAsString": true
       }
     },
+    "DiffDiskPlacement": {
+      "type": "string",
+      "description": "Specifies the ephemeral disk placement for operating system disk. This property can be used by user in the request to choose the location i.e, cache disk or resource disk space for Ephemeral OS disk provisioning. By default For more information on Ephemeral OS disk size requirements, please refer : https://docs.microsoft.com/en-us/azure/virtual-machines/windows/ephemeral-os-disks#size-requirements",
+      "enum": [
+        "CacheDisk",
+        "ResourceDisk"
+      ],
+      "x-ms-enum": {
+        "name": "DiffDiskPlacement",
+        "modelAsString": true
+      }
+    },
     "DiffDiskSettings": {
       "properties": {
         "option": {
           "$ref": "#/definitions/DiffDiskOption",
           "description": "Specifies the ephemeral disk settings for operating system disk."
+        },
+        "placement": {
+          "$ref": "#/definitions/DiffDiskPlacement",
+          "description": "Specifies the ephemeral disk placement for operating system disk. This property is used to specify Cache disk or Resource disk for ephemeral OS disk provisioning. By default if customer does not specify this placement property in the request, the Ephemeral OS disk will be provisioned using Cache disk."
```

Comment:
Default: Ephemeral OS disk will be placed in the CacheDisk if one is configured for the VM size otherwise ResourceDisk is used, refer to VM size documentation for Windows VM at https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes and Linux VM at https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes to check which VM sizes exposes a cache disk.